scripts/source-maintenance.sh

```diff
@@ -85,27 +102,38 @@ done
 # an error code seen by a KeepGoing-aware command (or zero)
 SeenErrors=0
 
-
 if ! git diff --quiet; then
 	echo "There are unstaged changes. This script may modify sources."
 	echo "Stage changes to avoid permanent losses when things go bad."
 	exit 1
 fi
 
-# On squid-cache.org we have to use the python scripted md5sum
-HOST=`hostname`
-if test "$HOST" = "squid-cache.org" ; then
-	MD5="md5"
+for Checksum in md5sum md5 shasum sha1sum
+do
+    if "$Checksum" </dev/null >/dev/null 2>/dev/null ; then
+        break
+    fi
+done
+echo "detected checksum program $Checksum"
+
+if [ "x$ASTYLE" != "x" ] ; then
+    if ${ASTYLE} --version >/dev/null 2>/dev/null ; then
+        :
+    else
+        echo "ERROR: cannot run user-supplied astyle ${ASTYLE}"
+    fi
 else
-	MD5="md5sum"
+    for AttemptedBinary in astyle-${TargetAstyleVersion} astyle
+    do
+        if $AttemptedBinary --version >/dev/null 2>/dev/null ; then
+            ASTYLE=$AttemptedBinary
+            echo "detected astyle program ${ASTYLE}"
+            break
+        fi
+    done
 fi
+export ASTYLE
```

Comment:
The binary is not yet fully confirmed to be the usable one. It should not be used to overwrite any existing environment variable yet.
```suggestion
```

Reply:
Now that scripts/formater.pl no longer receives ASTYLE environment variable the export can simply be removed.